.github/workflows/ci.yml

```diff
@@ -9,47 +9,40 @@ on:
-          - { path: python, name: python_slow, ci_node_total: 5, ci_node_index: 2 }
-          - { path: python, name: python_slow, ci_node_total: 5, ci_node_index: 3 }
-          - { path: python, name: python_slow, ci_node_total: 5, ci_node_index: 4 }
-          - { path: pub, name: pub, ci_node_total: 2, ci_node_index: 0 }
-          - { path: pub, name: pub, ci_node_total: 2, ci_node_index: 1 }
-          - { path: terraform, name: terraform }
+          - { path: bundler, name: bundler1, ci_node_total: 2, ci_node_index: 0, ecosystem: bundler }
+          - { path: bundler, name: bundler1, ci_node_total: 2, ci_node_index: 1, ecosystem: bundler }
+          - { path: bundler, name: bundler2, ci_node_total: 2, ci_node_index: 0, ecosystem: bundler }
+          - { path: bundler, name: bundler2, ci_node_total: 2, ci_node_index: 1, ecosystem: bundler }
+          - { path: cargo, name: cargo, ecosystem: cargo }
+          - { path: common, name: common, ecosystem: common}
+          - { path: composer, name: composer, ci_node_total: 2, ci_node_index: 0, ecosystem: composer }
+          - { path: composer, name: composer, ci_node_total: 2, ci_node_index: 1, ecosystem: composer }
+          - { path: docker, name: docker, ecosystem: docker }
+          - { path: elm, name: elm, ecosystem: elm }
+          - { path: git_submodules, name: git_submodules, ecosystem: submodule }
+          - { path: github_actions, name: github_actions, ecosystem: github-actions }
+          - { path: go_modules, name: go_module, ci_node_total: 2, ci_node_index: 0, ecosystem: gomod }
+          - { path: go_modules, name: go_module, ci_node_total: 2, ci_node_index: 1, ecosystem: gomod }
+          - { path: gradle, name: gradle, ecosystem: gradle }
+          - { path: hex, name: hex, ci_node_total: 2, ci_node_index: 0, ecosystem: hex }
+          - { path: hex, name: hex, ci_node_total: 2, ci_node_index: 1, ecosystem: hex }
+          - { path: maven, name: maven, ecosystem: maven }
+          - { path: npm_and_yarn, name: npm_and_yarn, ci_node_total: 3, ci_node_index: 0, ecosystem: npm }
+          - { path: npm_and_yarn, name: npm_and_yarn, ci_node_total: 3, ci_node_index: 1, ecosystem: npm }
+          - { path: npm_and_yarn, name: npm_and_yarn, ci_node_total: 3, ci_node_index: 2, ecosystem: npm }
+          - { path: nuget, name: nuget, ecosystem: nuget }
+          - { path: pub, name: pub, ci_node_total: 2, ci_node_index: 0, ecosystem: pub }
+          - { path: pub, name: pub, ci_node_total: 2, ci_node_index: 1, ecosystem: pub }
+          - { path: python, name: python, ci_node_total: 2, ci_node_index: 0, ecosystem: pip }
+          - { path: python, name: python, ci_node_total: 2, ci_node_index: 1, ecosystem: pip}
+          - { path: python, name: python_slow, ci_node_total: 2, ci_node_index: 0, ecosystem: pip }
+          - { path: python, name: python_slow, ci_node_total: 2, ci_node_index: 1, ecosystem: pip }
```

Comment:
Oh, one more question: Why is the number of python runs cut from 4 to 2?

Reply:
@pavera any reason for that?

Reply:
I suspect it's because @pavera sped up a lot the Python suite when he started pre-building python versions at `docker build` time, so now these are fast enough to use just two nodes.